app/javascript/dashboard/components/widgets/conversation/bubble/ImageAudioVideo.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="message-text__wrap" :class="attachmentTypeClasses">
+    <img
```

Comment:
Instead using image directly here, Can we use `<bubble-image` and `bubble-video` here?

Reply:
Yes, will change

Reply:
@nithindavid We cannot use `<bubble-image/>` and `<bubble-video/>` because we now use a different attachment view component with key shortcuts and arrow buttons. In `<bubble-image/>` and `<bubble-video/>`, a modal exists to view a single image or video.

Reply:
Makes sense. I'll test this.

Reply:
Thanks